paasta_tools/paasta_remote_run.py

```diff
@@ -37,18 +44,34 @@
 
 def parse_args(argv):
     parser = argparse.ArgumentParser(description='')
-    add_remote_run_args(parser)
-    parser.add_argument(
+    subs = parser.add_subparsers(dest='action', help='Subcommands of paasta_remote_run')
+
+    start_parser = subs.add_parser('start', help='Start task')
+    add_start_args_to_parser(start_parser)
+    add_common_args_to_parser(start_parser)
+    start_parser.add_argument(
         '-X', '--constraints-json',
         help=('Mesos constraints JSON'),
         required=False,
         default=None,
     )
+
+    stop_parser = subs.add_parser('stop', help='Stop task')
+    add_common_args_to_parser(stop_parser)
+    stop_parser.add_argument(
+        '-F', '--framework-id',
```

Comment:
I was kind hoping that we *didn't* need another thing that users have to know. 

Can we get away with just "instance" and have that dynamically calculated?

For example if a user launches "yelp-main.mycool-job"
I think they should be able to stop it with "paasta stop yelp-main.mycool-job", *not* have to run "paasta local-run stop my-cool-job -F frameworkid"

Is this possible?

I guess it is difficult because the date string and run_id are now part of the framework, we can't easily calculate it.

How about an alternative: I'm sure we have a function to list the frameworks, can we use the "chooser" function we have in paasta and allow people to interactively choose the framework id? (default to None, use the completer= function to get tab completion, etc)